<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dforget</class>
 <widget class="QWidget" name="Dforget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget
{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="backloginBtn">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/C:/Users/fdog/Desktop/lib/blackButton_2.jpg);</string>
       </property>
       <property name="text">
        <string>返回登录</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/pic/back.png</normaloff>:/pic/back.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SureBtn">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/C:/Users/fdog/Desktop/lib/blackButton_2.jpg);</string>
       </property>
       <property name="text">
        <string>确认修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;方正黑体简体&quot;;</string>
       </property>
       <property name="text">
        <string>验证码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sureEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="verifiBtn">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/C:/Users/fdog/Desktop/lib/blackButton_2.jpg);</string>
       </property>
       <property name="text">
        <string>发送验证码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;方正黑体简体&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/pic/name.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="accountLabel"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;方正黑体简体&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/pic/mima.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newpasswdLabel"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;方正黑体简体&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/pic/email.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
